src/entities/Member.ts

```diff
@@ -24,6 +25,11 @@ export class Member {
   @Column({ type: 'varchar' })
   password: string
 
+  @BeforeInsert()
+  async encodeString() {
+    this.password = await BcryptUtil.encodeString(this.password)
+  }
+
```

Comment:
@BeforeInsert annotation will trigger this encodeString method when inserting new member.

To run encodeString method while updating you should use `@BeforeUpdate` annotation.

Reply:
```suggestion
  @BeforeInsert()
  @BeforeUpdate()
  async encodeString() {
    this.password = await BcryptUtil.encodeString(this.password)
  }

```